oap-server/server-receiver-plugin/skywalking-trace-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/trace/provider/TraceServiceModuleConfig.java

```diff
@@ -41,7 +44,13 @@
     @Setter
     @Getter
     private int sampleRate = 10000;
-
+    /**
+     * Some of the agent can not have the upstream real network address, such as https://github.com/apache/skywalking-nginx-lua.
+     * service instance mapping and service instance client side relation are ignored.
+     */
+    @Setter
+    @Getter
+    private List<String> noUpstreamRealAddressAgents = Collections.emptyList();
```

Comment:
This should be initialized to include LUA. As a native agent, we know, LUA agent can't support in this value.

Reply:
done